src/controllers/confirm.company.controller.ts

```diff
@@ -41,6 +41,8 @@ export const post = async (req: Request, res: Response, next: NextFunction) => {
 const displayEligibilityStopPage = (res: Response, eligibilityStatusCode: EligibilityStatusCode) => {
   if (eligibilityStatusCode === EligibilityStatusCode.INVALID_COMPANY_STATUS) {
     return res.render(Templates.INVALID_COMPANY_STATUS);
+  } else if (eligibilityStatusCode === EligibilityStatusCode.INVALID_COMPANY_TYPE_USE_WEB_FILING) {
```

Comment:
now we're adding more pages we can create a 'mapping' object to save us having all the ifs.

you could add an object to hold the mapping like this:
```
const stopPages = {
  [EligibilityStatusCode.INVALID_COMPANY_STATUS]: Templates.INVALID_COMPANY_STATUS,
  [EligibilityStatusCode.INVALID_COMPANY_TYPE_USE_WEB_FILING]: Templates.USE_WEBFILING
};
```

Then you can change the displayEligibilityStopPage function to get the template from the stopPages object
```
const displayEligibilityStopPage = (res: Response, eligibilityStatusCode: EligibilityStatusCode) => {
  const stopPage: string = stopPages[eligibilityStatusCode];
  if (!stopPage) {
    throw new Error(`Unknown eligibilityStatusCode ${eligibilityStatusCode}`);
  }
  return res.render(stopPage);
};
```

Reply:
also need to add a test to check the USE_WEBFILING stop screen gets returned